src/components/Cards/AnimalGalleryCard.js

```diff
@@ -40,7 +40,7 @@ const AnimalGalleryCard = ({ animal, selectable, onSelectAnimal, onClickAnimal,
         setAnimalImage(animal.pictureUri ? animal.pictureUri : placeholder);
     }, [animal.type, animal.pictureUri, animalImage]);
 
```

Comment:
@TeresitaNader Good catch. Looks like I shouldn't have removed this class. I'll add that back in a second.